src/proximity_matcher_webservice/run_proximity_matcher_server.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 - Armijn Hemel
+# SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
```

Comment:
Not sure that this makes sense.

Reply:
> Thanks for the quick review! I modified the copyright.

yes, but not exactly what I meant! This is a new file, so not sure I should be mentioned in it. I also assume that the rights holder in this case would be TNG and not you personally. In that case the old format is much more correct (don't ask me, ask the German IT lawyers) ;-)

Reply:
I see. The copyright is modified accordingly.